packages/tailwind/README.md

```diff
@@ -3,3 +3,43 @@
 ```
 yarn add @kaizen/tailwind
 ```
+
+## Caveat before use
+
+This is an early release of the Kaizen Tailwind Preset - the API may change and evolve as we receive feedback from our teams. We will endeavour to assist where possible if classes or utilities are updated in the future.
+
+## Getting started
+
+>The assumption is that you have previously followed the FE Foundation guide for setting Tailwind up in your project.
+
+Install `@kaizen/tailwind` as a dependency and require it into your tailwind config. Then, pass it into your `presets` array.
+
+```js
+// this originally lived in @kaizen/desing-tokens but we've decoupled this for ease of versioning
+const { Preset } = require("@kaizen/tailwind")
+
+module.exports = {
+  content: ["./**/*.{ts,tsx}"],
+  presets: [Preset],
+  // important to add to the #root and #docs-root (for storybook) to ensure that tailwind classes supersede component styles
+  important: ["#root", "#docs-root"],
+  theme: {
+    extend: {},
+  },
+  plugins: [],
+}
+```
+
+> For preset configuration options, see [CONFIGURATION.md](CONFIGURATION.md)
+
+---
+
+## What our package does
```

Comment:
Somehow I feel like this might be good at the top of the doc where the caveat is. You can actually add a menu of sorts to the top to jump around in the file :)

Or pull the caveat down into here. Either way, so long as we can group the Install with the Getting Started so that the technical stuff is together.